index.d.ts

```diff
@@ -1,8 +1,15 @@
+interface Options {
```

Comment:
It needs to be wrapped in a `namespace`, per https://github.com/sindresorhus/typescript-definition-style-guide#documentation